src/main/java/uk/ac/ebi/ena/webin/cli/service/SampleService.java

```diff
@@ -47,32 +60,85 @@
         }
     }
 
-    private static class SampleResponse {
-        public int taxId;
-        public String id;
-        public String organism;
-        public String bioSampleId;
-        public boolean canBeReferenced;
+    /**

```

Comment:
```
 /**
Retrieves a sample from BioSamples or ENA. If the given sample ID is a BioSample 
accession then the sample information is retrieved from BioSamples. Otherwise, 
the sample information is retrieved from ENA. Also, if the sample is not found from 
BioSamples, then we will attempt to retrieve it from ENA.
@param sampleId the BioSample or SRA sample accession.
@return sample information from BioSample or ENA.
*/
```